processor/attributesprocessor/factory.go

```diff
@@ -115,12 +120,22 @@ func buildAttributesConfiguration(config Config) ([]attributeAction, error) {
 
 		switch a.Action {
 		case INSERT, UPDATE, UPSERT:
-			if a.Value == nil && a.FromAttribute == "" {
-				return nil, fmt.Errorf("error creating \"attributes\" processor. Either field \"value\" or \"from_attribute\" setting must be specified for %d-th action of processor %q", i, config.Name())
+			if a.Value != nil && a.Regex != "" {
+				return nil, fmt.Errorf("error creating \"attributes\" processor. Only of field \"value\" or \"regex\" setting must be specified for %d-th action of processor %q", i, config.Name())
```

Comment:
Nit: to be consistent with the similar error message below
```suggestion
				return nil, fmt.Errorf("error creating \"attributes\" processor due to both fields \"value\" and \"regex\" being set at the %d-th actions of processor %q", i, config.Name())
```

Reply:
Updated